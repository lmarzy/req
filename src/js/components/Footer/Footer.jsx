import React from 'react';

const Footer = () => (
  <footer>
    <small>&copy; 2017 React Kickstarter</small>
  </footer>
);

export default Footer;
