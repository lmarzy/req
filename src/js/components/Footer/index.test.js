import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from './index';

const component = shallow(<Footer />);
const tree = renderer.create(<Footer />).toJSON();

describe('<Footer />', () => {
  it('Should render 1 <Footer /> component', () => {
    expect(component).toHaveLength(1);
  });

  it('Render a snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
