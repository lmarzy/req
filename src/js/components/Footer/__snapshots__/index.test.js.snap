// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Render a snapshot 1`] = `
<footer>
  <small>
    Â© 
  </small>
</footer>
`;
